authentik/providers/saml/processors/assertion.py

```diff
@@ -136,10 +137,25 @@ def get_attributes(self) -> Element:
                 continue
         return attribute_statement
 
+    def _get_issuer_value(self) -> str:
+        """Get issuer value, with fallback to generated URL if empty"""
+        # If user has set an override issuer, use it
+        if self.provider.issuer:
+            return self.provider.issuer
+
+        # Otherwise, build off of request if application is linked
+        application = getattr(self.provider, "application", None)
```

Comment:
this check shouldn't be required as we get here from a PolicyAccessView which checks that this is set